examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Line #6.    configure_tensorflow()</p><p> </p><p>are you sure you want to use configure_tensorflow() it automatically takes 50% of the GPU memory..  instead you could use os.environ["TF_GPU_ALLOCATOR"]="cuda_malloc_async". </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>